drivers/scheduler/k8s/k8s.go

```diff
@@ -458,17 +459,21 @@ func (k *k8s) waitForCleanup(ctx *scheduler.Context, podList []v1.Pod) error {
 
 func (k *k8s) validateVolumeDirCleanup(podUID types.UID, app *spec.AppSpec) error {
 	podVolDir := k.getVolumeDirPath(podUID)
-	options := node.ConnectionOpts{
-		Timeout:         1 * time.Minute,
-		TimeBeforeRetry: 10 * time.Second,
-	}
 	driver, _ := node.Get(k.nodeDriverName)
+	options := node.FindOpts{
+		ConnectionOpts: node.ConnectionOpts{
+			Timeout:         1 * time.Minute,
+			TimeBeforeRetry: 10 * time.Second,
+		},
+		MinDepth: 1,
+		MaxDepth: 1,
+	}
 
 	for _, n := range k.GetNodes() {
 		if n.Type == node.TypeWorker {
-			if exists, err := driver.CheckIfPathExists(podVolDir, n, options); err != nil {
+			if volDir, err := driver.FindFiles(podVolDir, n, options); err != nil {
 				return err
-			} else if exists {
+			} else if strings.TrimSpace(volDir) != "" {
```

Comment:
need to include `volDir` in the error message so it propagates up explicitly in the logs.

Reply:
the volDir is /var/lib/kubelet/pods/<pod_uid>/volumes/*
, where the only imp thing is the pod UID

Reply:
Ok. I thought it has the volume name too. If not, no change is needed.